### DISCUSSION TITLE
Ability to group series in legend

### DISCUSSION THREAD

**Discussion by adhaussy** [grafana/grafana#38106] [Dashboards, Panels & Visualizations]
Hi,

Thanks to grafana features, i can mix several series in one panel, while still maintaining a pretty clean outlook. 
This is very powerfull, because it shows me a lot of relevant informations at a glance.

However, when mixing different series like for ex. Http codes, Go routines, Error types.. it can quickly become confusing when having a look at the legend.

It would be great if i could group series in the legend (like Stack A/B/C/D) to clear things up a bit (while still being able to use sort order.)

Let me show you an example :

![image](https://user-images.githubusercontent.com/16417068/130211423-1d38bbf7-688c-41c2-b6d3-3d60d49b4a88.png)

And something i'd like in a perfect world :

![image](https://user-images.githubusercontent.com/16417068/130211436-4569f606-d441-48de-ad6a-a1f551777b43.png)

thank you !

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
